ocm/__main__.py

```diff
@@ -255,20 +288,64 @@ def main():
     create_parser.set_defaults(callable=create)
 
     add_parser = maincmd_parsers.add_parser(
+        'add',
+    )
+    add_subparsers = add_parser.add_subparsers(
+        title='add ocm references',
+        required=True,
+    )
+    add_cref_parser = add_subparsers.add_parser(
+        'component-ref',
+        help='add component-reference',
+    )
+
+    add_cref_parser.add_argument('--file', '-f', required=True)
+    add_cref_parser.add_argument(
+        '--name', '-n',
+        help='name of the component-reference, defaults to last part of component-name',
+    )
+    add_cref_parser.add_argument(
+        '--component-name', '-c',
+        dest='cname',
+        required=True,
+        help='might optionally contain `:<version>`-suffix',
+    )
+    add_cref_parser.add_argument(
+        '--component-version', '-v',
+        dest='version',
+    )
+    add_cref_parser.add_argument(
```

Comment:
can't we merge this into the `append`-subcommand?

Reply:
well, you asked for a "dedicated command" and gave an example with `ocm add component-ref`

we could also incorporate into `append` command